luminoth/models/ssd/ssd_feature_extractor.py

```diff
@@ -0,0 +1,130 @@
+import tensorflow as tf
```

Comment:
Remove the `ssd_` at the start of the file, it's already prefixed by the package.